superset-frontend/package.json

```diff
@@ -129,7 +129,6 @@
     "d3-scale": "^2.1.2",
     "dnd-core": "^2.6.0",
     "dom-to-image": "^2.6.0",
-    "dompurify": "^2.0.7",
```

Comment:
the place where i removed this was the last instance of it, so i've removed it from package.json